src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftWriter.scala

```diff
@@ -264,8 +265,9 @@ private[redshift] class RedshiftWriter(
 
     // Convert all column names to lowercase, which is necessary for Redshift to be able to load
     // those columns (see #51).
-    val schemaWithLowercaseColumnNames: StructType =
-      StructType(data.schema.map(f => f.copy(name = f.name.toLowerCase)))
+    val schemaWithLowercaseColumnNames: StructType = if enableCaseSensitiveIdentifier == true
```

Comment:
nitpicking to rename this value since it won't always be lower case now.  Maybe `caseAdjustedSchema` or something like that?